hack/create-s3-bucket.sh

```diff
@@ -6,29 +6,33 @@ BUCKET_NAME=$1
 
 REGION=$(aws configure get region)
 
-create_bucket() {
+recreate_bucket() {
     if [[ "$REGION" = "us-east-1" ]]; then
+        aws s3api delete-bucket --bucket "$BUCKET_NAME" || true
         aws s3api create-bucket \
             --acl private \
-            --bucket $BUCKET_NAME \
-            --region $REGION
+            --bucket "$BUCKET_NAME" \
+            --region "$REGION"
     else
+        aws s3api delete-bucket --bucket "$BUCKET_NAME" -region "$REGION" || true
         aws s3api create-bucket \
             --acl private \
-            --bucket $BUCKET_NAME \
-            --region $REGION \
-            --create-bucket-configuration LocationConstraint=$REGION
+            --bucket "$BUCKET_NAME" \
+            --region "$REGION" \
+            --create-bucket-configuration LocationConstraint="$REGION"
     fi
 }
 
 block_public_access_to_bucket() {
     aws s3api put-public-access-block \
-        --bucket $BUCKET_NAME \
-        --public-access-block-configuration "BlockPublicAcls=true,IgnorePublicAcls=true,BlockPublicPolicy=true,RestrictPublicBuckets=true"
+        --bucket "$BUCKET_NAME" \
+        --region "$REGION" \
```

Comment:
There isn't a region flag for this command.

Reply:
https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-options.html 

From what I see region option is global CLI and should be available to all commands ...